src/lib/useRoller.ts

```diff
@@ -43,7 +108,244 @@ export default function useRoller() {
+          : [];
+
+      return points;
+    },
+    [api]
+  );
+
+  const getPendingTransactions = useCallback(async () => {
+    try {
+      const curPoint = need.point(pointCursor);
+      const newPending = await api.getPendingByShip(Number(curPoint));
+      setPendingTransactions(newPending);
+
+      // const allTransactions = await api.getHistory()
+    } catch (error) {
+      console.warn('ERROR GETTING PENDING', error);
+    }
+  }, [api, setPendingTransactions, pointCursor]);
+
+  const getInvites = useCallback(
+    async (isL2: boolean) => {
+      try {
+        const curPoint: string = need.point(pointCursor);
+        const invites = getStoredInvites(curPoint);
+        const availableInvites = invites.available;
+
+        const pendingTransactions = await api.getPendingByShip(curPoint);
+        console.log('PENDING', pendingTransactions);
+        setPendingTransactions(pendingTransactions);
+
+        const stillPending = invites.pending.filter(invite => {
+          const completed = !pendingTransactions.find(
+            p => p?.rawTx?.tx?.tx?.data?.ship === invite.planet
+          );
+
+          if (completed) {
+            availableInvites.push({ ...invite, status: 'available' });
+          }
+
+          return !completed;
+        });
+
+        setStoredInvites(curPoint, {
+          available: availableInvites,
+          pending: stillPending,
+          claimed: invites.claimed,
+        });
+        setInvites(availableInvites);
+
+        const _authToken = authToken.getOrElse(null);
+        const _contracts = contracts.getOrElse(null);
+
+        if (_authToken && _contracts) {
+          let possibleMissingInvites: number[] = [];
+          if (isL2) {
+            const allSpawned = await api.getSpawned(Number(curPoint));
+            const ownedPoints = maybeGetResult(
+              controlledPoints,
+              'ownedPoints',
+              []
+            );
+            possibleMissingInvites = allSpawned.filter(
+              (p: number) =>
+                ownedPoints.includes(p) &&
+                azimuth.azimuth.getPointSize(p) ===
+                  azimuth.azimuth.PointSize.Planet
+            );
+          } else {
+            const outgoingPoints = getOutgoingPoints(
+              controlledPoints,
+              getDetails
+            );
+
+            const availablePoints = await azimuth.azimuth.getUnspawnedChildren(
+              _contracts,
+              curPoint
+            );
+
+            possibleMissingInvites = outgoingPoints.filter(
+              (p: number) =>
+                azimuth.azimuth.getPointSize(p) ===
+                  azimuth.azimuth.PointSize.Planet &&
+                availablePoints.includes(p)
+            );
+          }
+
+          // Iterate over all spawned and controlled planets
+          // If the planet is not in available invites, generate the ticket and add it
+          console.log('POSSIBLE MISSING', possibleMissingInvites);
```

Comment:
These log statements seem useful for debugging, perhaps we should keep these behind a flag for future development?